doc improvement: aws_fota: Extend troubleshooting section (#14331)
nrfconnect/sdk-nrf: doc/nrf/libraries/networking/aws_fota.rst

   To fix this problem, configure the host server to provide this field.
   Also, confirm that your file is available from the browser without being logged into your AWS account.

Presigned URLs
--------------

When using presigned URLs it might be the needed to increase the value of:
 - :kconfig:option:`CONFIG_DOWNLOAD_CLIENT_MAX_FILENAME_SIZE`
 - :kconfig:option:`CONFIG_AWS_FOTA_PAYLOAD_SIZE`

>  - :kconfig:option:`CONFIG_MBEDTLS_HEAP_SIZE` - If running MBED TLS on the application core. (Wi-Fi builds)

Reviewer: ```suggestion
 - :kconfig:option:`CONFIG_MBEDTLS_HEAP_SIZE` - If running MbedTLS on the application core. (Wi-Fi builds)
```

Reviewer 2: Actually, Mbed TLS.

> When using presigned URLs it might be the needed to increase the value of:
>  - :kconfig:option:`CONFIG_DOWNLOAD_CLIENT_MAX_FILENAME_SIZE`
>  - :kconfig:option:`CONFIG_AWS_FOTA_PAYLOAD_SIZE`
>  - :kconfig:option:`CONFIG_MBEDTLS_HEAP_SIZE` - If running MBED TLS on the application core. (Wi-Fi builds)
> to accommadate for the increased file name and payload size of the presigned URL, and secure download of the image.

Reviewer 3: ```suggestion
When using the presigned URLs, you might need to increase the value of the following Kconfig options to accommodate the long file name and payload size of the presigned URL and the secure download of the image:

 - :kconfig:option:`CONFIG_DOWNLOAD_CLIENT_MAX_FILENAME_SIZE`.
 - :kconfig:option:`CONFIG_AWS_FOTA_PAYLOAD_SIZE`.
 - :kconfig:option:`CONFIG_MBEDTLS_HEAP_SIZE` - If running MBED TLS on the application core (Wi-Fi® builds).
```
This needs to be moved under the [Configuration](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/aws_fota.html#configuration) section.
